draft-ietf-quic-tls.md

```diff
@@ -1421,19 +1429,23 @@ amplification.
 
 ## Header Protection Analysis {#header-protect-analysis}
 
-Header protection relies on the packet protection AEAD being a pseudorandom
-function (PRF), which is not a property that AEAD algorithms
-guarantee. Therefore, no strong assurances about the general security of this
-mechanism can be shown in the general case. The AEAD algorithms described in
-this document are assumed to be PRFs.
-
-The header protection algorithms defined in this document take the form:
+{{NAN}} analyzes authenticated encryption algorithms which provide nonce
+privacy, referred to as "Hide Nonce" (HN) transforms. The general header
+protection construction in this document is one of those algorithms (HN1).
+Header protection uses the output of the packet protection AEAD to derive
+`sample`, and then encrypts the header field using a pseudorandom function
+(PRF) as follows:
 
 ~~~
 protected_field = field XOR PRF(hp_key, sample)
 ~~~
 
-This construction is secure against chosen plaintext attacks (IND-CPA) {{IMC}}.
+Assuming hp_key is distinct from the packet protection key, this construction
+(HN1) achieves AE2 security and therefore guarantees privacy of `field`, the
```

Comment:
AE2 security is not an easily digestible term on its own. What is the objective, where is it defined?

Reply:
It’s defined in the referenced paper. I thought it best to simply point readers in that direction if they want the details, rather try and pull it out here. (We didn’t previously define IND-CPA security, for example.)

Reply:
Even with citing, AE2 is not searchable. It is probably AE-scheme II, but I have only skimmed the paper.

Reply:
> We  give  a  modified  syntax  for  nonce-based encryption, called NBE2, in which decryption does not get the nonce, a corresponding framework of security definitions called AE2 that guarantee nonce privacy in addition to authenticity and message privacy, and simple ways to turn NBE1 AE1-secure schemes into NBE2 AE2-secure schemes.

This is in the introduction of the paper, under "Contributions in brief."

Reply:
Uh sorry, I was locked on to Rogaway's paper.